youtube_dl/extractor/vvvvid.py

```diff
@@ -172,12 +189,20 @@ def metadata_from_url(r_url):
                     if kenc_message:
                         embed_code += '?' + ds(kenc_message)
                 formats.extend(self._extract_akamai_formats(embed_code, video_id))
+            elif video_type == 'video/youtube':
+                info.update({
+                    '_type': 'url_transparent',
+                    'ie_key': YoutubeIE.ie_key(),
+                    'url': embed_code,
+                })
+                formats = None
             else:
                 formats.extend(self._extract_wowza_formats(
                     'http://sb.top-ix.org/videomg/_definst_/mp4:%s/playlist.m3u8' % embed_code, video_id))
             metadata_from_url(embed_code)
 
-        self._sort_formats(formats)
+        if formats:
+            self._sort_formats(formats)
```

Comment:
`_sort_formats` should always be called for non youtube videos(to break the extraction with a proper message when no formats has been extracted).

Reply:
https://github.com/ytdl-org/youtube-dl/pull/27825/commits/c5966a4dca65001c23540dc1dff2926dc869787d